docs/MachineConfiguration.md

```diff
@@ -113,6 +113,47 @@ spec:
 
 Note that for 4.2 clusters this is only supported as a "day 2" operation.
 
+#### Known issue
+On a 4.2 based OCP cluster if we already have kernel arguments applied using MachineConfig and then we try to create a new node using openshift-machine-api, existing kargs won't get applied. This behaviour is because 4.2 doesn't know how to process kernel arguments during firstboot on a newly spun node. See [bug#1766346](https://bugzilla.redhat.com/show_bug.cgi?id=1766346) for more information.
+
+Best way to fix this issue is by getting bootimage updated to 4.3 or later version which can be tracked at [enhancement proposal](https://github.com/openshift/enhancements/pull/201/) .
+
+Until we have the enhancement implemented it can be done manually by updating bootimage in machineset.
+
+**Example:**
+
+Suppose we have a 4.2 cluster created on aws and have applied karg `foo` in cluster with following MachineConfig -
+```
+$ cat worker-karg.yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: worker-kargs
+spec:
+  kernelArguments:
+    - foo
+```
+First step is upgrade cluster to OCP 4.3 or later version. Once the cluster has been bee successfully upgraded, list the available machinesets.
+```
+$ oc get machineset -n openshift-machine-api
+NAME                                          DESIRED   CURRENT   READY   AVAILABLE   AGE
+ci-ln-bx6fgqt-d5d6b-89fn7-worker-us-east-1b   2         2         2       2           79m
+ci-ln-bx6fgqt-d5d6b-89fn7-worker-us-east-1c   1         1         1       1           79m
+```
+
+In our example, let's edit the bootimage in machineset ci-ln-bx6fgqt-d5d6b-89fn7-worker-us-east-1c with corresponding 4.3 bootimage. For 4.4, list of bootimage for different platforms and regions are available [here](https://github.com/openshift/installer/blob/release-4.4/data/data/rhcos.json), checkout to respective OCP release branch in repo for corresponding release.  Since the cluster was created in aws and us-east-1 region, we will update ami id to [ami-0543fbfb4749f3c3b](https://github.com/openshift/installer/blob/release-4.4/data/data/rhcos.json#L43) with -
```

Comment:
Combining the above ideas:

```suggestion
In our example, let's edit the bootimage in machineset ci-ln-bx6fgqt-d5d6b-89fn7-worker-us-east-1c with corresponding 4.3 bootimage. For 4.4, a list of bootimages for different platforms and regions are available [here](https://github.com/openshift/installer/blob/release-4.4/data/data/rhcos.json), checkout the respective OCP release branch in the repo for the corresponding release.  Since the cluster was created in AWS in the us-east-1 region, we will update the ami id to [ami-0543fbfb4749f3c3b](https://github.com/openshift/installer/blob/release-4.4/data/data/rhcos.json#L43) with the following command:
```